src/AccessibilityInsights.Core/Results/RuleResult.cs

```diff
@@ -55,6 +51,16 @@ public class RuleResult
         /// </summary>
         public HelpUrl HelpUrl { get; set; }
 
+        /// <summary>
+        /// Returns the display text for the issue.
+        /// </summary>
+        public string IssueDisplayText { get; set; }
```

Comment:
@jalkire as I remember we don't need to return this result in the GitHub case?

Reply:
Yes, this string would not be populated if a github issue was filed